customer_portal_sessions.go

```diff
@@ -0,0 +1,80 @@
+// Code generated by the Paddle SDK Generator; DO NOT EDIT.
+
+package paddle
+
+import "context"
+
+// CustomerPortalSessionGeneralUrLs: Authenticated customer portal deep links that aren't associated with a specific entity.
+type CustomerPortalSessionGeneralUrLs struct {
+	// Overview: Link to the overview page in the customer portal.
+	Overview string `json:"overview,omitempty"`
+}
+
+/*
+CustomerPortalSessionSubscriptionUrLs: List of generated authenticated customer portal deep links for the subscriptions passed in the `subscription_ids` array in the request.
+
+If subscriptions are paused or canceled, links open the overview page for a subscription.
+
+Empty if no subscriptions passed in the request.
+*/
+type CustomerPortalSessionSubscriptionUrLs struct {
```

Comment:
```suggestion
type CustomerPortalSessionSubscriptionUrls struct {
```

Reply:
looks like this is a generation issue, we can see `SubscriptionManagementUrLs` also exists already with this

I'll manually fix these so when we fix we don't cause a BC, will follow up with a fresh generation fixing this across the codebase for v3

Good spot 🦅 👁️